cvmfs/network/jobinfo.h

```diff
@@ -98,6 +129,9 @@ class JobInfo {
     if (pipe_job_results.IsValid()) {
       pipe_job_results.Destroy();
     }
+    if (data_tube_.IsValid()) {
+      data_tube_.Destroy();
+    }
```

Comment:
This check is part of `Destroy()`
```suggestion
    data_tube_.Destroy();
```